tmt/steps/provision/testcloud.py

```diff
@@ -440,10 +441,10 @@ def _guess_image_url(self, name: str) -> str:
                 matched_fedora_coreos.group(2), self.arch)
         elif matched_centos[0]:
             url = testcloud.util.get_centos_image_url(
-                matched_centos[0].group(2), self.arch)
+                matched_centos[0].group(2), False, self.arch)
         elif matched_centos[1]:
             url = testcloud.util.get_centos_image_url(
-                matched_centos[1].group(2), stream=True, arch=self.arch)
+                matched_centos[1].group(2), True, self.arch)
```

Comment:
Why drop parameter names here and on line 444 while adding them on line 427?

Reply:
I just followed the previous form, I'll keep them there then.

Reply:
My bad, the previous form had the parameter names, I mixed it up with the python testcloud script itself.